packages/core/src/endowmentsToolkit.js

```diff
@@ -751,6 +751,12 @@ function defaultCreateFunctionWrapper(sourceValue, unwrapTest, unwrapTo) {
     newValue,
     Object.getOwnPropertyDescriptors(sourceValue)
   )
+  // Global functions can have static fields on a prototype. eg. Uint8Array.from
+  if (
+    Reflect.getPrototypeOf(newValue) !== Reflect.getPrototypeOf(sourceValue)
+  ) {
+    Object.setPrototypeOf(newValue, Reflect.getPrototypeOf(sourceValue))
```

Comment:
Is `Object.setPrototypeOf` necessary instead of `Reflect.setPrototypeOf`?  Does it matter?  If it does, please explain.